### DISCUSSION TITLE
error: Manual upgrade required. Expected file format version 109, but file is version 107

### DISCUSSION THREAD

**Discussion by therealshulgin** [ordinals/ord#1627] [General]
When running `./ord wallet inscription` I get `error: Manual upgrade required. Expected file format version 109, but file is version 107`

Anyone knows what's going wrong ?

**Comment by raphjaph** [comment]
Did you compile ord manually from `master`?

We made some changes to the database on the `master` branch, so the old  database won't work. You can build a new database by deleting the old `index.redb` or create a new one by building a database in a specified directory with `ord --data-dir <LOCATION> index`. If you do the latter you will have to provide `--data-dir` flag for each `ord` command.

**Reply by MaxflowO2** [reply]
where is index.redb?

**Reply by raphjaph** [reply]
Depends on your system.
Mac: `~/Library/Application\ Support/ord/`
Linux: `~/.local/share/ord/`
Windows:`~\AppData\Roaming\ord`

**Reply by MaxflowO2** [reply]
awesome thank you, couldn't find it with `find ./ index.redb` in ord directory.

**Comment by therealshulgin** [comment]
Will do that thanks.

**Comment by liuser** [comment]
Why do we need to reindex with each version update? It takes me three days each time to complete it.

**Reply by raphjaph** [reply]
because implementing migrations for every version is a lot of work.
